apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Store, select } from '@ngrx/store';
 import { Observable } from 'rxjs';
 
 import { ToggleSidebar, CloseSidebar, OpenSidebar, SetSidebarState } from '../../actions/sidebar.actions';
 import * as fromDaffioSidebar from '../../reducers/index';
 
 @Component({
-  selector: 'sidebar-viewport-container',
+  selector: 'daffio-sidebar-viewport-container',
   templateUrl: './sidebar-viewport.component.html'
 })
-export class SidebarViewportContainer {
+export class DaffioSidebarViewportContainerComponent implements OnInit{
```

Comment:
I think this should be named `DaffioSidebarViewportContainer`.

Reply:
resolved in dbcf3deb24b3190d5c086b952e576b6e7fca991d